### PR TITLE
Update SunPy Project ideas

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: _projects/2020/sunpy/fido.md [hunk 1]
```diff
@@ -15,12 +15,14 @@ issues:
...
  - Cadair
  - nabobalis
+ - astrofrog
 initiatives:
  - GSOC
 tags:
  - python
 collaborating_projects:
  - SunPy
+ - Glue
```
Comment (nabobalis) [inline _projects/2020/sunpy/fido.md]: Wrong one?

File: _projects/2020/sunpy/fido.md [hunk 2]
```diff
@@ -15,12 +15,14 @@ issues:
 mentors:
  - Cadair
  - nabobalis
+ - astrofrog
```
Comment (nabobalis) [inline _projects/2020/sunpy/fido.md]: Wrong one?

File: _projects/2020/sunpy/fido.md (hunk 1 above)
Comment (Cadair) [reply _projects/2020/sunpy/fido.md]: balls

**Comment by bsipocz** [comment]
Merge away when you're ready. Also please merge mine.

**Pull request merged**